draft-ietf-rats-corim.md

```diff
@@ -1599,12 +1599,14 @@ The `addition` is added to the ACS for a specific Attester.
 {::include cddl/intrep-ae.cddl}
 ~~~
 
-| Type | `en` | `el` | `a` | `cm` | `p` |
+| ECT type  | ECT Field       | Requirement |
 |---
-| addition | T | T | T | T | F |
-{: #tbl-ae-ect-optionality title="Mandatory fields for Evidence tuples"}
-
-'T' means mandatory.
+| addition  | `environment`   | Mandatory   |
+|           | `elements`      | Mandatory   |
+|           | `authority`     | Mandatory   |
+|           | `cm type`       | Mandatory   |
```

Comment:
```suggestion
|           | `cm type`       | Mandatory (2)  |
```
https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/297

Shall we prescribe the cmtype based on the internal representation?

Reference values' cmtype must be 0. Endorsed values and conditional endorsed value additions must be 1. Policy must be 5...?

Reply:
> #297
> 
> Shall we prescribe the cmtype based on the internal representation?

Yes. 

> 
> Reference values' cmtype must be 0. Endorsed values and conditional endorsed value additions must be 1. Policy must be 5...?

I don't think I follow. The table is saying the internal representation will populate cmtype. The structure of cm-type is defined in the ECT definition section.

Reply:
I think your transformation PR accounts for the values to populate cmtype with.